platforms/cli/build.gradle.kts

```diff
@@ -6,23 +6,23 @@ val javaMainClass = "com.dfsek.terra.cli.TerraCLI"
 
 dependencies {
     shadedApi(project(":common:implementation:base"))
-    
-    shadedApi("commons-io:commons-io:${Versions.CLI.commonsIO}")
+
+    shadedApi("commons-io:commons-io:${Versions.Libraries.Internal.apacheIO}")
```

Comment:
Why are we not doing `shadedApi("commons-io", "commons-io", Versions.Libraries.Internal.apacheIO)` here?

Reply:
Because ci is never really touched. resolved